tests/bgp/test_ipv6_nlri_over_ipv4.py

```diff
@@ -0,0 +1,197 @@
+'''
+
+This script is to verify DUT's ability to carry both IPv4 and IPv6
+Network Layer Reachability Information (NLRI) over a single IPv4 BGP session.
+
+'''
+import logging
+
+import pytest
+import time
+# import re
+
+from natsort import natsorted
+
+logger = logging.getLogger(__name__)
+# dut_4byte_asn = 400003
+# neighbor_4byte_asn = 400001
+
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.device_type('vs')
```

Comment:
This device type line would live this testcase to run only on vs testbed. You might want to remove this line.

Reply:
This has been removed.